packages/frontend/types/emberclear/index.d.ts

```diff
@@ -93,6 +93,7 @@ declare global {
   interface ChannelMember {
     id: string;
     name: string;
+    signingKey: string;
```

Comment:
public or private? (I assume public, based on context)
no need to rename, but a comment might be helpful

Reply:
It is public. I will add in a comment.

Reply:
Addressed in b8fb4e8